app/assets/javascripts/components/meetings_list.es6.jsx

```diff
@@ -0,0 +1,68 @@
+class MeetingsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.perPage = 12;
+    this.meetings = [];
```

Comment:
Why are you storing meetings as a local variable? Wouldn't it be safer to store it in the component's local state?